app/src/main/java/org/fossasia/susi/ai/chat/adapters/viewholders/PieChartViewHolder.java

```diff
@@ -66,13 +68,15 @@ public void setView(ChatMessage model) {
                 pieChart.setRotationAngle(0);
                 pieChart.setDragDecelerationFrictionCoef(0.001f);
                 pieChart.getLegend().setEnabled(false);
-                pieChart.setDescription("");
+                Description description = new Description();
+                description.setText("");
+                pieChart.setDescription(description);
                 RealmList<Datum> datumList = model.getDatumRealmList();
-                final ArrayList<Entry> yVals = new ArrayList<>();
+                final ArrayList<PieEntry> yVals = new ArrayList<>();
                 final ArrayList<String> xVals = new ArrayList<>();
                 for (int i = 0; i < datumList.size(); i++) {
-                    yVals.add(new Entry(datumList.get(i).getPercent(), i));
-                    xVals.add(datumList.get(i).getPresident());
+                    yVals.add(new PieEntry(datumList.get(i).getPercent(),
+                            datumList.get(i).getPresident()));
```

Comment:
I think the space here should be removed.

Reply:
@batbrain7 That is not an extra space. That is another parameter to the PieEntry constructor, hence that automatically indents on pressing enter.

Reply:
okay sorry I mistook it for the green rectange usually the space

Reply:
🙂